src/psyclone/tests/psyir/nodes/omp_directives_test.py

```diff
@@ -50,7 +50,8 @@
     OMPTaskwaitDirective, OMPTargetDirective, OMPLoopDirective, Schedule, \
     Return, OMPSingleDirective, Loop, Literal, Routine, Assignment, \
     Reference, OMPDeclareTargetDirective, OMPNowaitClause, \
-    OMPGrainsizeClause, OMPNumTasksClause, OMPNogroupClause
+    OMPGrainsizeClause, OMPNumTasksClause, OMPNogroupClause, CodeBlock, \
+    BinaryOperation, Assignment
```

Comment:
pylint points out that Assignment is imported twice.

Reply:
Fixed